tests/test_create_model.py

```diff
@@ -516,6 +518,53 @@ def test_create_model_non_annotated():
         create_model('FooModel', foo=(str, ...), bar=123)
 
 
+@pytest.mark.skipif(sys.version_info < (3, 9), reason='Annotated is introduced after python3.9')
+@pytest.mark.parametrize(
+    'annotation_type,field_info',
+    [
+        (bool, Field(alias='foo_bool_alias', description='foo boolean')),
+        (str, Field(alias='foo_str_alis', description='foo string')),
+    ],
+)
+def test_create_model_typing_annotated_field_info(annotation_type, field_info):
+    annotated_foo = typing.Annotated[annotation_type, field_info]
+    model = create_model('FooModel', foo=annotated_foo, bar=(int, 123))
+
+    assert model.model_fields.keys() == {'foo', 'bar'}
+
+    foo = model.model_fields.get('foo')
+
+    assert foo is not None
+    assert foo.annotation == annotation_type
+    assert foo.alias == field_info.alias
+    assert foo.description == field_info.description
+
+
+@pytest.mark.skipif(sys.version_info < (3, 9), reason='Annotated is introduced after python3.9')
+def test_create_model_typing_annotated_field_uses_first_type():
+    annotated_foo = typing.Annotated[int, 10, str]
```

Comment:
I think the way to specify a default value should be:
`Annotated[int, Field(default=10)]`

`Annotated[int, 10]` is not supported for normal Pydantic models afaik so it would be weird to support it for `create_model`.

Reply:
The Annotated in this case should be essentially equivalent to `(int, 10)`, which in the current use case it will set the field annotation as `int`. 

The second input variable would be simply a metadata of the annotated field and the interpretation seems to be left for the implementation. If the consistency seems to be the problem I would suggest we change the normal model accept that case as well. 

I focused more on the consistency between the usage of tuple as input since I feel like restricting users from using metadata alone may be a bit restrictive.

Reply:
I agree with @Viicos, I think we should keep the default specification consistent and go with solely the `Annotated[int, Field(default=10)]` approach.

Reply:
Sounds good, in that case let me specify the input metadata to contain `FieldInfo`